src/maps/components/markers.js

```diff
@@ -10,7 +11,8 @@ export function groupMarker (group) {
     fontIcon: group.isMember ? 'fas fa-home' : 'fas fa-info-circle',
     color: group.isMember ? 'positive' : 'blue',
     popup: {
-      render: h => h(GroupMarker, { props: { group } }),
+      component: markRaw(GroupMarker),
```

Comment:
I wonder why this needed to be marked raw? Maybe an explanatory comment?

Reply:
```suggestion
      component: markRaw(GroupMarker),  // markers are part of reactive data, but don't want the component itself to become reactive (otherwise some error happens)
```